scripts/sysdig-deploy/update-sysdig-deploy.sh

```diff
@@ -13,24 +14,30 @@ minor=0
 major=0
 patch=0
 
+# helper functions to preserve empty lines and avoid unnecessary changes
+bump_dep_preserve_blank() {
+    sed -i'' "$(yq '( .dependencies[] | select(.name == "'"$1"'") | .version) | line' "charts/$4/Chart.yaml")s/$2/$3/" "charts/$4/Chart.yaml"
```

Comment:
Wondering what makes `-i''` necessary instead of a plain `-i`
If it's a quirk of some sort, it should be spelled out explicitly before someone like my future self thinks of removing it. :-)

Reply:
added comment